src/c1_foeppl_von_karman/foeppl_von_karman_curvable_bell_elements.cc

```diff
@@ -0,0 +1,93 @@
+// LIC// ====================================================================
+// LIC// This file forms part of oomph-lib, the object-oriented,
+// LIC// multi-physics finite-element library, available
+// LIC// at http://www.oomph-lib.org.
+// LIC//
+// LIC//    Version 1.0; svn revision $LastChangedRevision: 1097 $
+// LIC//
+// LIC// $LastChangedDate: 2015-12-17 11:53:17 +0000 (Thu, 17 Dec 2015) $
+// LIC//
+// LIC// Copyright (C) 2006-2016 Matthias Heil and Andrew Hazel
+// LIC//
+// LIC// This library is free software; you can redistribute it and/or
+// LIC// modify it under the terms of the GNU Lesser General Public
+// LIC// License as published by the Free Software Foundation; either
+// LIC// version 2.1 of the License, or (at your option) any later version.
+// LIC//
+// LIC// This library is distributed in the hope that it will be useful,
+// LIC// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// LIC// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// LIC// Lesser General Public License for more details.
+// LIC//
+// LIC// You should have received a copy of the GNU Lesser General Public
+// LIC// License along with this library; if not, write to the Free Software
+// LIC// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+// LIC// 02110-1301  USA.
+// LIC//
+// LIC// The authors may be contacted at oomph-lib@maths.man.ac.uk.
+// LIC//
+// LIC//====================================================================
+// Non--inline functions for BellFvK elements
+#include "foeppl_von_karman.h"
+
+namespace oomph
+{
+  //=============================================================================
+  /// Set the number of fields
+  //=============================================================================
+  template<unsigned NNODE_1D>
+  const unsigned FoepplVonKarmanC1CurvableBellElement<NNODE_1D>::Nfield = 3;
+
+
+  //=============================================================================
+  /// Set the interpolation of each field
+  //=============================================================================
+  template<unsigned NNODE_1D>
+  const std::vector<bool> FoepplVonKarmanC1CurvableBellElement<
+    NNODE_1D>::Field_is_bell_interpolated = {false, false, true};
+
+
+  //======================================================================
+  /// Set the data for the number of Variables at each node
```

Comment:
explain. I think the w variables are only used at the vertex nodes the midside (or other) nodes are only used for the two in plane displ. right?